packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -324,6 +324,18 @@ export function createOdspNetworkError(
 				);
 				break;
 			}
+		case 429: // Throttling error
+			// TODO: check !FeatureFlag value here to opt-out of this behavior.
+			if(props.storageServiceError){
```

Comment:
>storageServiceError [](http://example.com/codeflow?start=12&length=19)

if it is not a storage error, ie, push errors, we do want the default behavior below, isn't it ?

Reply:
In the code I am actually letting the default condition handle the relay service scenario. Is you recommendation to explicitly add logic for non-storage errors as well within the case 429?